### PR TITLE
feat: deploy dec 13 batch 

### PR DESCRIPTION
**Pull request by paulbalaji**
### Description

Deploy to new chains: arthera, aurora, conflux, conwai, corn, evmos, form, ink, rivalz, soneium, sonic, telos.

### Drive-by changes

- igp updates
- fix minor bug in setting up infra multiprovider with secrets
	- ensures we're able to fetch secrets for chains not yet in `supportedChainNames` e.g. for checking balances on new chains before adding them to infra

### Testing

manual

### CODE REVIEW

File: .registryrc [hunk 1]
```diff
@@ -1 +1 @@
-01f2271592da4a06962e31ed384201f32cb070ab
+e09afa103441c1692f87c33deb0ccf1df1f01fc0
```
Comment (paulbalaji) [inline .registryrc:1]: update after merging https://github.com/hyperlane-xyz/hyperlane-registry/pull/431

File: typescript/infra/config/environments/mainnet3/agent.ts [hunk 2]
```diff
@@ -223,19 +245,23 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     rarichain: true,
     real: true,
     redstone: true,
+    rivalz: true,
```
Comment (daniel-savu) [inline typescript/infra/config/environments/mainnet3/agent.ts:248]: rivalz wasn't in the PR description but assuming it is part of the new batch

File: typescript/infra/scripts/check/check-validator-version.ts [hunk 3]
```diff
@@ -15,27 +15,33 @@ import { getEnvironmentConfig, getHyperlaneCore } from '../core-utils.js';
...
   // Rolled out only to AW infra before 1.0.0, just 1 commit behind 1.0.0
-  'a64af8be9a76120d0cfc727bb70660fa07e70cce': 'pre-1.0.0',
+  'a64af8be9a76120d0cfc727bb70660fa07e70cce': '1.0.0-beta',
   // 1.0.0
   'ffbe1dd82e2452dbc111b6fb469a34fb870da8f1': '1.0.0',
-  // Tessellated's Own Build
-  '9b855686d3e2b3d6b81238ce51a576ff5e0f770f': 'tesselated-special-build',
+  // Tessellated's Build from November 2024
+  // https://github.com/Tessellated-io/hyperlane-monorepo/commit/9b855686d3e2b3d6b81238ce51a576ff5e0f770f
+  '9b855686d3e2b3d6b81238ce51a576ff5e0f770f': 'Tesselated - November 2024',
```
Comment (daniel-savu) [inline typescript/infra/scripts/check/check-validator-version.ts:41]: what's this?

**Review by daniel-savu — APPROVED** [review]

File: typescript/infra/config/environments/mainnet3/agent.ts (hunk 2 above)
Comment (paulbalaji) [reply typescript/infra/config/environments/mainnet3/agent.ts:248]: good catch - i updated changeset but not description

**Pull request merged**